src/odemis/gui/cont/tabs/cryo_chamber_tab.py

```diff
@@ -683,6 +714,20 @@ def _on_aligner_btn(self, evt):
         self._show_warning_msg(None)
         self.panel.btn_cancel.Enable()
 
+    def _edit_meteor_positions(self, evt):
+        """Open a dialog for editing meteor stage positions and metadata"""
+
+        stage_md = self._stage.getMetadata()
+        dialog = EditMeteorPositionsDialog(self.main_frame, stage_md)
+        dialog.Center()
+
+        _ = dialog.ShowModal()
+        if dialog._update_pressed:
```

Comment:
Depending on a private variable is not the right way. The typical way in wxPython is to check the return value of `ShowModal`. If it something like `wx.OK`, then it means the user expects the data to be used.
Alternatively, you could have a special value of `stage_md` when the user didn't accept the changes.